docs/getting-started.md

```diff
@@ -52,28 +52,32 @@ There are three repos relevant to building on The Graph:
 Below we outline the required steps to build a subgraph from scratch, that will be serving up queries from a GraphQL endpoint. There are three major steps:
 
 1. Define The Subgraph
+    1. [Create a subgraph project and manifest](#1-defining-the-subgraph)
 
-    1. [Create a subgraph project and manifest](#3-defining-the-subgraph)
-    
-    2. [Define data sources](#31-define-the-data-sources-and-create-a-manifest)
-    
-    3. [Define a GraphQL schema](#32-create-the-graphql-schema-for-the-data-source)
-    
-    4. [Generate types for use in the subgraph mappings](#33-graph-cli)
-    5. [Write mappings](#34-writing-mappings)
+    2. [Define data sources](#11-define-the-data-sources-and-create-a-manifest)
+
+    3. [Define a GraphQL schema](#12-create-the-graphql-schema-for-the-data-source)
+
+    4. [Generate types for use in the subgraph mappings](#13-setup-graph-cli-project-and-generate-types)
+
+    5. [Write mappings](#14-writing-mappings)
 2. Build and Deploy The Subgraph
-    1. [Build and deploy mappings to IPFS](#41-build-and-deploy-the-mappings-to-ipfs)
-    
-    1. [Deploy the subgraph to the local Graph Node](#42-deploying-the-subgraph)
+    1. [Start up an IPFS node](#21-start-up-ipfs)
+
+    2. [Install Postgres and setup a database](#22-create-the-postgres-db)
+
+    3. [Starting The Graph Node](#23-starting-the-graph-node-and-connecting-to-an-etheruem-node)
```

Comment:
This is the only bullet point that uses a gerund. Can we change it to `Start the Graph Node`?